src/db/db_backup.py

```
@@ -13,42 +11,50 @@
+        "-U",
+        settings.db_user,
+        "-d",
+        settings.db_name,
+        "-f",
+        str(backup_path),
+        "--no-owner",
+        "--clean",
+    ]
+
+    try:
+        subprocess.run(command, env=env, check=True, capture_output=True, text=True)
```

Ruff (0.15.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)